### ISSUE TITLE
Make it easy to view the HTML report

### ISSUE THREAD

**Issue by hammer** [hammerlab/epidisco#9]
According to @ihodes:

> the idea now though is to move the report to gbuckets and serve it off there

**Comment by smondet** [comment]
Another option (for temporary viewing for example):

build the cluster with:

```
export WITH_WEBSERVER=true
```

and then on `$PREFIX-pbs-server`:

```
cd /var/www/html/
ln -s /path/to/results  some-cool-name
```

**Comment by ihodes** [comment]
Alternative solution to this in #15

**Comment by smondet** [comment]
Yes #15 solves this, cf. the demo: https://storage.googleapis.com/protoepidisco-results/training-demo/index.html
but it is not yet automated (I'm not sure how to automate it nicely, since we need to run `gsutil` from an account that has the right rights configured).

**Comment by ihodes** [comment]
closed in favor of #15
